### DISCUSSION TITLE
请求废都物语

### DISCUSSION THREAD

**Discussion by linshb** [Gamuxorg/bbs#198] [投递和请求游戏]
游戏名称：废都物语
游戏名称(英文)：
下载地址 :（如果有的话）：http://pan.baidu.com/share/link?shareid=132763&uk=3406177925

**Comment by linshb** [comment]
这是个好游戏，我想要在Linux上也能玩。可惜wine解决不了因为日文乱码而启动不了游戏的问题。所以求大佬帮忙打包一下。

**Comment by wangling12** [comment]
我用纯净的容器跑了一下
![2020-09-22 21-21-39 的屏幕截图](https://user-images.githubusercontent.com/24770600/93887719-a3a91b80-fd19-11ea-9738-7adddf3b8add.png)
如果你的结果和我一样的话，这不是日语，也不是乱码，只是分辨率太低导致的
那几个应该是开始游戏，继续游戏和退出游戏。。。
